lib/vanagon/platform/windows.rb

```diff
@@ -8,10 +8,13 @@ class Windows < Vanagon::Platform
       def generate_package(project)
         # If nothing is passed in as platform type, default to building a nuget package
         # We should default to building an MSI once that code has been implimented
+        # SUGGEST WE TAKE DEFAULT OUT ALTOGETHER - WE MUST HAVE A package_type DEFINED on Windows
```

Comment:
Have rebased to changes in #258 - Had to resolve a conflict over the generate_package/package_name methods.
Also removed comments w.r.t. default as they are no longer relevant here.